Mathlib/RingTheory/IntegralRestrict.lean

```diff
@@ -0,0 +1,129 @@
+/-
+Copyright (c) 2023 Andrew Yang, Patrick Lutz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.RingTheory.IntegrallyClosed
+import Mathlib.RingTheory.Norm
+import Mathlib.RingTheory.DedekindDomain.IntegralClosure
+/-!
+# Restriction of various maps between fields to integrally closed subrings.
+
+## Main definition
+- `galRestrict`: The restriction `Aut(L/K) → Aut(B/A)` as an `MulEquiv` in an ALKB setup.
```

Comment:
```suggestion
- `galRestrictHom`: The restriction `Aut(L/K) → Aut(B/A)` as an `MulEquiv` in an ALKB setup.
```
And what does ALKB mean?

Reply:
I guess it means A a subring of L and B a subring of K?

Reply:
`galRestrict` is the main definition to use. Updated the docstring of `galRestrictHom` to make it clearer.
Also added a sentence in the module docstring explaining AKLB.